src/ros_filter.cpp

```diff
@@ -1035,13 +1036,26 @@ void RosFilter<T>::loadParams()
     "set_pose", rclcpp::QoS(1),
     std::bind(&RosFilter<T>::setPoseCallback, this, std::placeholders::_1));
 
+  // Create a subscriber for manually setting/resetting state
+  set_state_sub_ =
```

Comment:
Do we want to support this via messages? This feels like a service call should suffice.